pupil_src/shared_modules/offline_surface_tracker.py

```diff
@@ -471,9 +471,9 @@ def save_surface_statsics_to_file(self, export_range, export_dir):
             surface_name = '_'+s.name.replace('/','')+'_'+s.uid
 
             #save surface_positions as csv
-            with open(os.path.join(metrics_dir,'srf_positons'+surface_name+'.csv'),'w',encoding='utf-8',newline='') as csvfile:
+            with open(os.path.join(metrics_dir,'surface_positons'+surface_name+'.csv'),'w',encoding='utf-8',newline='') as csvfile:
                 csv_writer =csv.writer(csvfile, delimiter=',')
-                csv_writer.writerow(('frame_idx','timestamp','m_to_screen','m_from_screen','detected_markers'))
```

Comment:
Are you sure that `m_to_screen` does not mean `surfce to original frame`? In this case the rename would be the other way around.

Reply:
My current understanding is that the matrix `m_to_screen` transforms points in scene image coordinates into points in normalized surface coordinates. Cant't say that I am sure though, I am still investigating the code!

Reply:
I can now confidently state that `m_to_screen` is mapping from surface coordinates to normalized image coordinates. I'd now propose to change the naming in accordance to what I use in #1268 : `surf_to_img_trans` or something similarly explicit.